terraform/modules/google/compute/haproxy/haproxy.tf

```diff
@@ -0,0 +1,55 @@
+#--------------------------------------------------------------
+# This module creates all resources necessary for HAProxy
+#--------------------------------------------------------------
+
+variable "name"              { }
+variable "zones" {}
+variable "atlas_username"    { }
+variable "atlas_environment" { }
+variable "atlas_token"       { }
+variable "private_subnet_names"              { }
+variable "image" {}
+variable "nodes"             { }
+variable "instance_type"     { }
+
+resource "template_file" "haproxy_config" {
```

Comment:
Converted all template_file resources to data sources, since count is supported as of 0.7.3 (cd9de66).